packages/components/pages/f-registration/package.json

```diff
@@ -58,9 +58,10 @@
     "@justeat/browserslist-config-fozzie": ">=1.1.1"
   },
   "devDependencies": {
-    "@justeat/f-button": "3.0.2",
-    "@justeat/f-card": "3.0.0",
-    "@justeat/f-error-message": "1.0.0",
+    "@justeat/fozzie": "9.0.0-beta.6",
+    "@justeat/f-button": "4.x",
```

Comment:
I may be missing something here, but what is the logic behind defining the version of the dependencies in such way?

Reply:
There's a PR which explains but in short the devDeps need to match the peerDep so yarn pulls down the correct versions. https://github.com/justeat/fozzie-components/pull/1956